utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/ui/GenerateTestsDialogWindow.kt

```
@@ -454,7 +454,9 @@ class GenerateTestsDialogWindow(val model: GenerateTestsModel) : DialogWrapper(m
             row {
                 cell(parametrizedTestSources)
                 contextHelp("Parametrization is not supported in some configurations, e.g. if mocks are used.")
-            }
+            }.enabledIf(ComboBoxPredicate(springConfig) {
+                model.projectType != ProjectType.Spring
```

It is better to write `model.projectType == ProjectType.PureJvm`
Getting Python here will hide an error of using incorrect dialog in your version.

---

This change should be applied everywhere, e.g. line 858.